<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardDialog</class>
 <widget class="QDialog" name="StandardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandardDialog</string>
  </property>
  <widget class="QLineEdit" name="fileLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="colorFrame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="fontLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fileBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>文件对话框</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colorBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色对话框</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fontBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>93</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>字体对话框</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
